components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/handlers/SessionDataPublisherImpl.java

```diff
@@ -0,0 +1,348 @@
+/*
+ *Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Licenses header must be formatted by adding a space between the `*` and the first letter of the sentence.

Reply:
resolved